diff --git a/bazel/grpc_deps.bzl b/bazel/grpc_deps.bzl
index 6752260f16..6108187095 100644
--- bazel/grpc_deps.bzl
+++ bazel/grpc_deps.bzl
@@ -55,26 +55,6 @@ def grpc_deps():
             ],
         )
 
-    if "com_google_protobuf" not in native.existing_rules():
-        http_archive(
-            name = "com_google_protobuf",
-            sha256 = "d0e3a75876a81e1536028bb9cf9181382b198da4cc6fa6aef86879ef629ac807",
-            strip_prefix = "protobuf-74211c0dfc2777318ab53c2cd2c317a2ef9012de",
-            urls = [
-                # https://github.com/protocolbuffers/protobuf/commits/v31.1
-                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/protocolbuffers/protobuf/archive/74211c0dfc2777318ab53c2cd2c317a2ef9012de.tar.gz",
-                "https://github.com/protocolbuffers/protobuf/archive/74211c0dfc2777318ab53c2cd2c317a2ef9012de.tar.gz",
-            ],
-            patches = [
-                "@com_github_grpc_grpc//third_party:protobuf.patch",
-            ],
-            patch_args = ["-p1"],
-            repo_mapping = {
-                "@abseil-cpp": "@com_google_absl",
-                "@googletest": "@com_google_googletest",
-            },
-        )
-
     if "com_google_googletest" not in native.existing_rules():
         http_archive(
             name = "com_google_googletest",
