diff --git a/google/spanner/admin/instance/v1/spanner_instance_admin.proto b/google/spanner/admin/instance/v1/spanner_instance_admin.proto
index 69043c1b..5630d653 100644
--- a/google/spanner/admin/instance/v1/spanner_instance_admin.proto
+++ b/google/spanner/admin/instance/v1/spanner_instance_admin.proto
@@ -368,6 +368,11 @@ message Instance {
   // for more information about nodes.
   int32 node_count = 5;

+  // The number of processing units allocated to this instance. At most one of
+  // processing_units or node_count should be present in the message. This may
+  // be zero in API responses for instances that are not yet in state `READY`.
+  int32 processing_units = 9;
+
   // Output only. The current instance state. For
   // [CreateInstance][google.spanner.admin.instance.v1.InstanceAdmin.CreateInstance], the state must be
   // either omitted or set to `CREATING`. For
